apps/site/site.json

```diff
@@ -3,7 +3,6 @@
   "description": "Node.js® is a free, open-source, cross-platform JavaScript runtime environment that lets developers create servers, web apps, command line tools and scripts.",
   "favicon": "/static/images/favicons/favicon.png",
   "accentColor": "#333",
-  "logoVariant": "pride",
```

Comment:
Can you use "défaut" I think it's work

Reply:
This is a direct inverse of the PR to add the Pride logo. Feel free to open a follow up after this lands

Reply:
Okay 👌